libraries/xcversion.rb

```diff
@@ -53,10 +53,11 @@ def installed_xcodes
       def available_versions
         lines = shell_out!(XCVersion.list_available_xcodes).stdout.lines
         lines.reject { |line| line.start_with?(/\D/) }
+        lines.map { |line| line.chomp(' (installed)') }
```

Comment:
Removing the extra space before `(installed)` seems a bit weird. Would trimming all leading/trailing whitespace make more sense?
```suggestion
        lines.map { |line| line.chomp('(installed)').strip }
```

Reply:
That's definitely more explicit and readable, it would be easy to miss the space in a refactor.